spec/lib/lighthouse/benefits_claims/service_spec.rb

```diff
@@ -266,6 +266,54 @@
           end
         end
       end
+
+      describe 'when submitting a 2122' do
+        let(:lh_config) { double }
+        let(:attributes) do
+          {
+            veteran: {
+              address: {
+                addressLine1: '936 Gus Points',
+                city: 'Watersborough',
+                countryCode: 'US',
+                stateCode: 'CO',
+                zipCode: '36090'
+              },
+              recordConsent: true,
+              consentLimits: %w[DRUG_ABUSE ALCOHOLISM HIV SICKLE_CELL]
+            },
+            serviceOrganization: {
+              poaCode: '095',
+              registrationNumber: '999999999999'
+            }
+          }
+        end
+        let(:expected_data) { { data: { attributes: } } }
+
+        context 'successful submit' do
+          it 'submits the correct data to lighthouse' do
+            allow(Common::Client::Base).to receive(:configuration).and_return lh_config
```

Comment:
This double is stopping a VCR-mediated request from being made. I think we would completely test this functionality with just using using VCR with strictest request matching and not doing the extra work of making these manual receive expectations.

You can see what is meant by "strictest request matching" here in an spec helper we're adding to the ARP module in an [upcoming PR](https://github.com/department-of-veterans-affairs/vets-api/pull/20752/files#diff-710c871dcf54dcc8fd9f5ea46ff159090603793489baf624f40f6954f10e5e42).

Since this is outside the ARP module (and that code wasn't introduced yet anyway), you could just merge in these VCR options:
```
{ match_requests_on: %i[method uri headers body] }
```

Then we can remove the double.